LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 MOBI Wireless Management
```

Comment:
Should this be "Tangoe" instead?

Reply:
Umm... I think so. Honestly, I took this one from one of our other open source gems. https://github.com/mobi/telephone_number